pkg/fleet-manager/fleet_plugin_backup.go

```diff
@@ -81,7 +81,7 @@ func (f *FleetManager) reconcileBackupPlugin(ctx context.Context, fleet *v1alpha
 	// Iterating through each fleet cluster to generate and apply Velero helm configurations.
 	for key, cluster := range fleetClusters {
 		// generate Velero helm config for each fleet cluster
-		b, err := plugin.RenderVelero(f.Manifests, fleetNN, fleetOwnerRef, plugin.FleetCluster{
+		b, err := plugin.RenderVelero(f.Manifests, fleetNN, fleetOwnerRef, plugin.RenderableFleetCluster{
```

Comment:
can we use the FleetCluster, if any field miss, we can add it

Reply:
the origin `FleetCluster`' fileld ' value is comes from two struct `fleetCluster`  and `ClusterKey` ,
so maybe we can not direct add filed in `FleetCluster`, or we need merge those two struct into one struct, which may affect many other code

https://github.com/kurator-dev/kurator/blob/2871c99124b0f27a3fcdd9542fb41db0b5cf1d89/pkg/fleet-manager/fleet_cluster.go#L34-L43

this is the most used usage:

https://github.com/kurator-dev/kurator/blob/83fdec03d1ed0b5f01c6e3d5ed5e47f038d4f5f6/pkg/fleet-manager/fleet_plugin_distributedstorage.go#L78-L81

 	FleetCluster.Name:       ClusterKey.Name, 
 	FleetCluster.SecretName: fleetCluster.Secret, 
 	FleetCluster.SecretKey:  fleetCluster.SecretKey,